doc/modules/model_persistence.rst

```diff
@@ -89,4 +83,28 @@ another architecture is not supported.
 
 If you want to know more about these issues and explore other possible
 serialization methods, please refer to this
-`talk by Alex Gaynor <https://pyvideo.org/video/2566/pickles-are-for-delis-not-software>`_.
+`talk by Alex Gaynor
+<https://pyvideo.org/video/2566/pickles-are-for-delis-not-software>`_.
+
+Interoperable formats
+---------------------
+
+For production and quality control needs, exporting the model in `Predictive
```

Comment:
I am thinking that we could be explicit that the model will only use the prediction part without the possibility to be refitted once exported.

Reply:
I have added a note at the beginning of the section: is that ok with you?

Reply:
Sorry, still puzzled about the necessity of retraining using a serialized model ... using the original python script seems a more transparent option to me especially if you look for architecture or environment dependencies...
I've modified the note as you suggested... but still not convinced to move it in the interoperable section....